src/features/users/user-route-validation.ts

```diff
@@ -134,5 +134,6 @@ export const adminUpdateValidation = {
         verified: Joi.boolean().optional(),
         firstName: Joi.string().optional(),
         lastName: Joi.string().optional(),
+        roleId: Joi.number().min(0).max(1).optional(),
```

Comment:
`allow([0,1])` would have been better so someone messing around couldn't pass `1.5`

They will just get a sequelize error so it doesn't really matter